docs/Configuration.md

```diff
@@ -363,6 +363,64 @@ Any public field or setter method in this class can be given a default value usi
 }
 ```
 
+## Routing modes
+
+The routing request parameter `mode` determines which transport modalities should be considered when calculating the list
+of routes.
+
+Some modes (mostly bicycle and car) also have optional qualifiers `RENT` and `PARK` to specify if vehicles are to be parked at a station or rented. In theory
+this can also apply to other modes but makes sense only in select cases which are listed below.
+
+Whether a transport mode is available highly depends on the input feeds (GTFS, OSM, bike sharing feeds) and the graph building options supplied to OTP.
+
+The complete list of modes are:
+
+- `WALK`: Walking some or all of the route.
+
+    _Prerequisite_: The maximum walking distance configured has not been exceeded.
```

Comment:
This prerequisite is not always true and is rather complex to explain why, so I'd just remove it altogether.